test/distributed/org/apache/cassandra/fuzz/harry/gen/EntropySourceTest.java

```diff
@@ -1,158 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.cassandra.fuzz.harry.gen;
-
-import java.util.Random;
-
-import org.junit.Assert;
-import org.junit.Test;
-
-import org.apache.cassandra.harry.gen.EntropySource;
-import org.apache.cassandra.harry.gen.distribution.Distribution;
-import org.apache.cassandra.harry.gen.rng.PCGFastPure;
-import org.apache.cassandra.harry.gen.rng.PcgRSUFast;
-import org.apache.cassandra.harry.model.OpSelectors;
-
-public class EntropySourceTest
```

Comment:
what test replaces this?

Reply:
The idea was that we will be switching to Accord EntropySource, so I did not port it. Brought it back for now.

Reply:
> switching to Accord EntropySource

You mean `RandomSource`?   since this patch doesn't include that scope I would avoid dropping tests for now

Reply:
Sure I brought them back.